web/packages/teleport/src/components/FormLogin/__snapshots__/FormLogin.story.test.tsx.snap

```diff
@@ -756,7 +756,7 @@ exports[`auth2faType: optional rendering 1`] = `
                           <div
                             class="react-select__single-value css-1uccc91-singleValue"
                           >
-                            Hardware Key
+                            Passkey or Security Key
```

Comment:
actually, for the FormLogin, doesn't `passkey` == `passwordless`? so `passkey or secuirty key` is saying `login with passwordless or security key`

but in the FormLogin there is a `passwordless` sign in button (rendered only if passwordless is allowed)

Reply:
oh nvm, i saw this `or other _allowed_ passkey type`